apollo-federation/tests/dhat_profiling/connectors_validation.rs

```diff
@@ -1,12 +1,16 @@
 #[global_allocator]
 pub(crate) static ALLOC: dhat::Alloc = dhat::Alloc;
 
+// Failure of the test can be diagnosed using the dhat-heap.json file.
+// Here is a write up on analysing dhat-heap.json:
+// https://apollographql.atlassian.net/wiki/spaces/Router/pages/1285226499/Using+dhat+to+profile+memory+utilisation+in+the+router#Analysing-dhat-heap.json
```

Comment:
nit: while it is great to have a link to this doc.... this a link to private docs so unsure whether that should be part of the commit in the public repo